lib/ubi_cli.rb

```diff
@@ -85,7 +85,7 @@ def self.list(cmd, fields)
       run do |opts|
         opts = opts[key]
         path = if (location = opts[:location])
-          if LocationNameConverter.to_internal_name(location)
+          if ProviderLocation[display_name: location]&.internal_name
             "location/#{location}/#{fragment}"
           else
             raise Rodish::CommandFailure, "invalid location provided in #{cmd} list -l option"
```

Comment:
```suggestion
          if location.include?("/")
            raise Rodish::CommandFailure, "invalid location provided in #{cmd} list -l option"
          else
            "location/#{location}/#{fragment}"
```

You can modify the test (or tests) that uses an invalid location to have the location include a `/`.  Other invalid locations will still result in an error during the internal API request.